media-gfx/blender/blender-3.2.2.ebuild

```diff
@@ -101,6 +102,7 @@ RDEPEND="${PYTHON_DEPS}
 	potrace? ( media-gfx/potrace )
 	pugixml? ( dev-libs/pugixml )
 	pulseaudio? ( media-sound/pulseaudio )
+	rocm? ( >=dev-util/hip-5.1.3 )
```

Comment:
I agree. There is another reason for me to change.

At first I use the hip flag. Then `pkgcheck` jumps out, because hip is amd64 only. Like cuda, I edited `use.mask` to mask this use flag for non-amd64.

Unlike cuda, which is the only name, some packages may use `rocm` (like [hwloc](https://github.com/gentoo/gentoo/pull/25837)), so we may need two entries, `rocm` and `hip`, in `use.mask`, while indicating almost the same thing (ROCm platform)

WDYT?

Reply:
I think you can have both `hip` and `rocm` in the same mask entry, yes.
So you can probably just have something like this:
```
 # ROCm (and HIP) is currently only supported on the amd64 arch
 rocm
 hip
 ```